lib/Installation/Popups/LicensePopup.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles License Popup.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Installation::Popups::LicensePopup;
+use parent 'YaST::PageBase';
+use strict;
+use warnings;
+use Installation::Popups::AcceptPopup;
```

Comment:
I got lost here, I don't understand why there is a popup inside another pop ...
Both are different containers, they might have some part repeated, but in this case doesn't worth to do composition.

Reply:
The idea was to create something like AbstractOKPopup.pm that could be used in other cases. But I assume that this is not found so often to worth a separate .pm